encoding/serialization.go

```diff
@@ -39,11 +44,15 @@ func Decode(b []byte) (Frame, error) {
 }
 
 func (c *G1Commitment) Serialize() ([]byte, error) {
-	return encode(c)
+	res := (*bn254.G1Affine)(c).Bytes()
+	return res[:], nil
 }
 
 func (c *G1Commitment) Deserialize(data []byte) (*G1Commitment, error) {
-	err := decode(data, c)
+	_, err := (*bn254.G1Affine)(c).SetBytes(data)
+	if err != nil {
+		return nil, err
+	}
 	return c, err
 }
 
```

Comment:
These changes seem unnecessary. However, since these methods only appear to be used for communication between batcher and encoder, I don't think there is a big impact.

Reply:
it is pointed out by the audit. Unless there is performance downgrade, it doesn't hurt to make it safe.